test/html_json.test.js

```diff
@@ -36,6 +39,17 @@ describe('HTML Indexing', () => {
 
   before(async () => {
     testRoot = await createTestRoot();
+    nock('https://raw.githubusercontent.com/')
+      .get(/\/helix\/github.com--adobe--helix-index-pipelines\/[^/]+\/helix-query.yaml/)
+      .replyWithFile(200, resolve(specsDir, 'blog', 'helix-query.yaml'))
+      .persist();
```

Comment:
maybe you should also test a case, where `helix-query.yaml` doesn't exist.

Reply:
It just returns an empty configuration then:
```
logger.error(`Failed to load index configuration from: ${url}`, e);
// config not usable return empty
return {
  indices: [],
};
```